nautobot/cloud/templates/cloud/cloudtype_retrieve.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load helpers %}
+
+{% block extra_breadcrumbs %}
+    <li><a href="{% url 'cloud:cloudtype_list' %}?provider={{ object.provider.pk }}">{{ object.provider }}</a></li>
+{% endblock extra_breadcrumbs %}
+
+{% block content_left_page %}
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>Cloud Type</strong>
+        </div>
+        <table class="table table-hover panel-body attr-table">
+            <tr>
+                <td>Name</td>
+                <td>
+                    {{ object.name }}
+                </td>
+            </tr>
+            <tr>
+                <td>Provider</td>
+                <td>{{ object.provider|hyperlinked_object }}</td>
+            </tr>
+            <tr>
+                <td>Content Type(s)</td>
+                <td>
+                    {% if object.content_types.all %}
+                        <ul>
+                            {% for content_type in object.content_types.all|dictsort:"model"|dictsort:"app_label" %}
```

Comment:
I'm surprised the `dictsort` are needed, aren't content-types normally ordered by app-label and model by default?